documentation/doxygen/Doxyfile

```diff
@@ -2254,12 +2254,7 @@ INCLUDE_FILE_PATTERNS  =
 # recursively expanded use the := operator instead of the = operator.
 # This tag requires that the tag ENABLE_PREPROCESSING is set to YES.
 
-PREDEFINED             = "ClassDef(x,y)=//" \
-                         ClassImp(x)=// \
-                         ClassImpQ(x)=// \
-                         templateClassImp(x)=// \
-                         NamespaceImp(x)=// \
```

Comment:
I'm convinced that we still want the `ClassImp`s to be hidden?

Reply:
What about marking instead `GenerateInitInstance`, `RegisterClassTemplate` as HIDDEN_SYMBOLS ?
I think it's more effective to mark that intention inline in the source code rather than in the Doxyfile. For example, if one decides to rename the function, it's easier to forget that the name has to be also updated in the Doxyfile.
But I can also change it back, it's not a big deal.

Reply:
> What about marking instead GenerateInitInstance, RegisterClassTemplate, etc. as HIDDEN_SYMBOLS ?

Probably fine.